src/Components/Web.Extensions/src/wwwroot/headManager.js

```diff
@@ -0,0 +1,51 @@
+// Local helpers
+
+const getHeadElement = (tagName, id) => {
```

Comment:
Any reason to use `const name = (...) => { ... }` instead of the more regular syntax?

```js
function name(...) {
   ...
}
```

Reply:
Not in particular. I've now updated it 😃